addons/website/static/src/components/configurator/configurator.js

```diff
@@ -518,6 +518,12 @@ class Store {
     }
 
     selectWebsitePurpose(id) {
+        // Keep track or the former selection in order to be able to keep
+        // the auto-advance navigation scheme while being able to use the
+        // browser's back and forward buttons.
+        if (!id && this.selectedPurpose) {
+            this.formerSelectedPurpose = this.selectedPurpose;
+        }
```

Comment:
That bug is also in 15.2, maybe two PR then? :+1:

Reply:
Problem actually appears in 15.0. Extracted to https://github.com/odoo/odoo/pull/96155.